mongodbatlas/clusters.go

```diff
@@ -395,6 +408,31 @@ func (s *ClustersServiceOp) GetProcessArgs(ctx context.Context, groupID, cluster
 	return root, resp, err
 }
 
+// Status gets the status of the operation on the Cluster.
+// See more: https://docs.atlas.mongodb.com/reference/api/clusters-check-operation-status/
```

Comment:
[nit] I have some pending work on fixing the other methods but this renders the wrong way on the go docs so let's not add to the noise
```suggestion
// Status gets the status of the operation on the Cluster.
//
// See more: https://docs.atlas.mongodb.com/reference/api/clusters-check-operation-status/
```